daal4py/sklearn/decomposition/pca.py

```diff
@@ -472,3 +472,22 @@ def transform(self, X):
             if self.whiten:
                 X_transformed /= np.sqrt(self.explained_variance_)
             return X_transformed
+
+if (lambda s: (int(s[:4]), int(s[6:])))( daal4py.__daal_link_version__[:8] ) < (2019, 4):
```

Comment:
does this mean we have PCA only for older DAAL versions?
A comment would be helpful...

Reply:
Yes, sure. With the older version of DAAL we have PCA where only fit has been optimized with DAAL's SVD call.

With DAAL 2019.4 `fit`, `transform` and `fit_transform` methods are optimized, and `svd_solver='daal'` is supported.

I will add the comment.